.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: tests
     strategy:
       matrix:
-        pyver: ["3.9", "3.10", "3.11"]
+        pyver: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
I added 3.12 to explicitly check this with the python version in the latest `rubin-env`. I left 3.9 as is, since it acts as an implicit check with `numpy<2`.

Reply:
SGTM!